components/messenger.rst

```diff
@@ -188,6 +235,6 @@ Receiver and Sender on the same bus
 
 To allow us to receive and send messages on the same bus and prevent an infinite
 loop, the message bus is equipped with the ``WrapIntoReceivedMessage`` middleware.
```

Comment:
Actually, the `WrapIntoReceivedMessage` is neither a middleware, neither it is used in Symfony core. That's the `Worker` that call the `receive` method with a `$handler`adding the `ReceivedMessage` marker to the envelope.

Hence I wondered multiple times if it was really intended? Should we remove the `WrapIntoReceivedMessage` decorator? Should we decorate in the core any created receiver? Should the worker only dispatch the received envelope to the bus, or keep adding `ReceivedMessage`?